src/components/RegistrationList.tsx

```diff
@@ -104,9 +104,11 @@ export const RegistrationListItemContent = ({
             columnGap: '1rem',
             whiteSpace: 'nowrap',
           }}>
-          <Box sx={{ display: 'flex', alignItems: 'center', columnGap: '1rem', flexWrap: 'wrap' }}>
+          <Box sx={{ display: 'flex', alignItems: 'center', columnGap: '0.5rem', flexWrap: 'wrap' }}>
             {focusedContributors.map((contributor, index) => (
-              <Box key={index} sx={{ display: 'flex', alignItems: 'center' }}>
+              <Box
+                key={index}
+                sx={{ display: 'flex', alignItems: 'center', '&:not(:last-child)': { '&:after': { content: '";"' } } }}>
```

Comment:
noen grunn til at dette er løst med CSS her, mens det de to andre tilfelle er lagt i en `span`? Vil oppføre seg bittelitt ulikt, da man ikke kan markere og kopiere `;` som legges til i CSS (det kan for så vidt være en bra ting akkurat her da)